### ISSUE TITLE
Color mismatch

### ISSUE THREAD

**Issue by TmLev** [Yazeed1s/minimal.nvim#4]
For some reason, colors of minimal-base16 don't match with those from the screenshots.

I'm sure that this issue has nothing to do with tmux/terminal/shell, since I've tried different environments.

| What I see | Screenshot from readme |
| -- | -- |
| <img width="673" alt="image" src="https://user-images.githubusercontent.com/22966523/188195736-e2d345bd-2480-48fe-8a59-c6fcba1b0332.png"> | ![image](https://user-images.githubusercontent.com/22966523/188195784-dc59ecd9-e728-4b29-adc7-9ca84f3ed2b9.png) |

What useful info can I provide?

**Comment by tminhvu** [comment]
Are you talking about the brackets? It's because of `nvim-ts-rainbow`. Mine caused the same issue too, red brackets. The highlights for `nvim-ts-rainbow`are `rainbowcol1` to `rainbowcol7`. Maybe you can modify those to your liking.

**Comment by TmLev** [comment]
Not exactly. Please take a look at `UnionFind`. Its name and types of fields should be red, but they are light green.

**Comment by nafis-bin** [comment]
I am getting the same issue the colors do not match the screenshots
